controllers/articles.controllers.js

```diff
@@ -1,9 +1,18 @@
 // articles.controllers.js
-const { selectArticleById } = require("../models/articles.models");
+const {
+  selectArticleById,
+  selectAllArticles,
+} = require("../models/articles.models");
 
 exports.getArticleById = (req, res, next) => {
   const { article_id } = req.params;
   selectArticleById(article_id)
     .then((article) => res.status(200).send({ article }))
     .catch(next);
 };
+
+exports.getAllArticles = (req, res, next) => {
+  selectAllArticles()
+    .then((articles) => res.status(200).send({ articles }))
+    .catch(next);
+};
```

Comment:
Controller looks great, glad you've also sent the object back on a descriptive key of articles